tensorboard/webapp/metrics/views/card_renderer/histogram_card_test.ts

```diff
@@ -298,7 +298,7 @@ describe('histogram card', () => {
         By.directive(TestableHistogramWidget)
       );
       expect(viz.componentInstance.linkedTime).toEqual({
-        start: {step: 5},
+        start: {step: 1},
```

Comment:
I am confused why this is 1. I am assuming that the closes step is 1 but, I do not see where the steps are set at all. Can we make that more obvious? Ideally we set the steps right here in this step and maybe make the steps around step 5 so this is more interesting. ex: [1, 2, 4, 8] and expect it to give 4.

Reply:
the data is set in "provideMockCardSeriesData" in testing.ts and the histogram data provided is [0, 1, 99] 
(same for scalar and image data, but we don't care here)
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/testing.ts;l=182?q=provideMockCardSeriesData&ss=tensorflow%2Ftensorboard

added a comment to help clarify